vulscan.nse

```diff
@@ -197,7 +197,7 @@ Download: http://www.computec.ch/mruef/?s=software&l=x
 
 --@thanks
 -- I would like to thank a number of people which supported me in
--- developing this script: Stefan Friedli, Simon Zumstein, Sean R�tschi,
+-- developing this script: Stefan Friedli, Simon Zumstein, Sean Rütschi,
```

Comment:
i didn't actually change this line, its identical on both as far as i can tell...